packages/react-app/src/context/CustomerContext.ts

```diff
@@ -0,0 +1,12 @@
+import { BANKLESS } from '../constants/Bankless';
+import { CustomerProps } from '../models/Customer';
```

Comment:
typings and context object for the customer context, the provider is initialised in the App component